trafilatura/utils.py

```diff
@@ -346,6 +346,20 @@ def trim(string: str) -> str:
         return ""
 
 
+def is_image_element(element: _Element):
```

Comment:
The return type should be `-> bool` right?

Reply:
ok, fixed